dnsdad.com.dnsdadrecords.json

```diff
@@ -0,0 +1,37 @@
+{
+    "providerId": "dnsdad.com",
+    "providerName": "dnsdad.com",
+    "serviceId": "dnsdadrecords",
+    "serviceName": "Dnsdad Records",
+    "version": 1,
+    "logoUrl": "https://dnsdad.com/logo.svg",
+    "description": "Configure DNS records using DNSDAD",
+    "variableDescription": "IP address for the domain A, CNAME. Domain for TXT, MX",
```

Comment:
Please review the variable description. It looks like not referring to the actual variables, like MX.

Reply:
Actually, we planned for MX Record earlier and later removed the MX record but forget to remove MX record from Description. I update the Description.